src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -560,13 +577,20 @@ object Helpers extends SparkSessionWrapper {
    * @return
    */
   def getWorkspaceByDatabase(
-                              etlDB: String,
+//                              etlDB: String,
+                              db: String,
```

Comment:
why did you change the name? it was etlDB so users no that this cannot reference the consumer DB

Reply:
I have derived etlDB through db params, so that I dont have to change this variable name through out the function. can I change db to etlDatabaseName ?

Reply:
done